src/transactions/transactions.service.ts

```diff
@@ -227,7 +227,7 @@ export class TransactionsService {
       };
 
       if (status === TransactionStatus.Succeeded) {
-        payload.result = 'placeholder'; // TODO @monitz87: use real result when we eliminate TQs
+        payload.result = 'placeholder'; // The SDK needs to support returning this (DA-407)
```

Comment:
@VictorVicente I think this is OK to leave as a placeholder for now. 

I opened up DA-407 with Jere's comments to track the change needed.

Reply:
I'm ok with the placeholder too.